crates/muvm/src/cli_options.rs

```diff
@@ -142,6 +143,10 @@ pub fn options() -> OptionParser<Options> {
         )
         .argument("COMMAND")
         .many();
+    let udevd_path = long("udevd-path")
```

Comment:
IMO this can just be a runtime env var. It's not important enough to need to be a command-line option.